var/spack/repos/builtin/packages/r-gmp/package.py

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RGmp(RPackage):
+    """Multiple Precision Arithmetic (big integers and rationals, prime
+       number tests, matrix computation), "arithmetic without limitations"
+       using the C library GMP (GNU Multiple Precision Arithmetic)."""
+
+    homepage = "http://mulcyber.toulouse.inra.fr/projects/gmp"
+    url      = "https://cran.r-project.org/src/contrib/gmp_0.5-12.tar.gz"
```

Comment:
Can you make sure that the URL here and the version number below match? It won't affect fetching, but it will affect my URL-parsing tests (try running `spack url summary` for more info).